samples/productSearch/productSets.js

```diff
@@ -134,6 +134,45 @@ async function deleteProductSet(projectId, location, productSetId) {
   // [END vision_product_search_delete_product_set]
 }
 
+async function purgeProductsInProductSet(projectId, location, productSetId) {
+  // [START vision_product_search_purge_products_in_product_set]
```

Comment:
Region tags should go on the outside of your function, per the sample policies so users can copy paste and don't run into async/await errors

Reply:
Thanks! Related to your previous comment....  I was doing it as if this were a main() function per sample standards since I'm used to writing them that way lol

Reply:
Lol and I am used to writing them the other way.